templates/zerver/emails/missed_message.txt

```diff
@@ -13,12 +13,13 @@ While you were away you received {{ message_count }} new{% if group_pm %} group
 {% endfor %}
 {% endif %}
 
-Click here to log in to Zulip and view your new messages:
-
-{{ realm_uri }}
-
+Reply to this email directly,
 {% if reply_to_zulip  %}
-Or just reply to this email.
+or view it in Zulip:
+{{ realm_uri }}
+{% else %}
+or Reply in Zulip:
+{{ realm_uri }}
```

Comment:
These changes correspond to the changes made in `missed_message.source.html` but in plain text format.